robot/Cumulus/resources/locators_48.py

```diff
@@ -254,6 +263,12 @@
         'new_button': '//button[text() = "{}"]',
         'select_locator': "//select[@name ='{}']",
         'success_toast': "//div[contains(@class,'slds-notify.slds-notify_toast,slds-theme_success')]",
+        'target_option': "//select[@name = 'Target Field']/option[text()='{}']",
+        'target': "//select[@name = 'Target Field']",
+        'target_object': "//select[@name = 'Target Object']",
```

Comment:
these could be changed to 1 by passing arguments in keyword, same for 266 and 269

Reply:
Actually realized I am not using them so deleted them